### ISSUE TITLE
Enhancement: add/bypass proxy

### ISSUE THREAD

**Issue by 2SI3NX** [ancestorak/brief#268]
Is it possible to add proxy settings in the general settings for the user to fill in a proxy, and if the user wishes to exclude a certain feed from using the proxy, you can have an additional tick-box setting in the settings of the feed saying "bypass proxy for this feed".
How difficult would this be to do?

**Comment by tanriol** [comment]
Do you mean that you want that your feeds are loaded through a proxy while the same websites are loaded directly when anything except the feed is loaded? I don't fully understand what you want to achieve...

**Comment by 2SI3NX** [comment]
Yes, in other words that the feeds can use a proxy like tor to update. and if you open the site it would load directly from the browser's normal connection.

**Comment by 2SI3NX** [comment]
and if one of the feeds doesnt update with the specified proxy, then you can set the specific feed in its settings to "bypass the proxy"
